src/app/app.module.ts

```diff
@@ -3,12 +3,16 @@ import { BrowserAnimationsModule } from '@angular/platform-browser/animations';
 import { NgModule } from '@angular/core';
 import { FormsModule, ReactiveFormsModule } from '@angular/forms';
 import { HttpModule } from '@angular/http';
-import { MaterialModule } from '@angular/material';
 import { FlexLayoutModule } from '@angular/flex-layout';
 import { AppRoutingModule } from './app-routing/app-routing.module';
 
-import { MdAutocompleteModule } from '@angular/material';
-import { MdSelectModule } from '@angular/material';
+
+import {MatAutocompleteModule} from '@angular/material';
+import {MatToolbarModule} from '@angular/material';
+import {MatFormFieldModule} from '@angular/material';
+import {MatCardModule} from '@angular/material';
+import {MatButtonModule} from '@angular/material';
+import {MatSelectModule} from '@angular/material';
```

Comment:
I think we could write this as: 
```
import {
MatAutocompleteModule, 
MatToolbarModule,
...
} from '@angular/material'
```
so that it's easier to know and maintain all the modules being imported from the same source.